src/main/java/com/google/api/generator/Main.java

```diff
@@ -37,7 +35,7 @@ public static void main(String[] args)
   }
 
   /** Register all extensions needed to process API protofiles. */
-  private static void registerAllExtensions(ExtensionRegistry extensionRegistry) {
+  public static void registerAllExtensions(ExtensionRegistry extensionRegistry) {
```

Comment:
Could we move this into a separate file? Having the main classes all call something like `Registry.registerAllExtensions` has more consistent semantics than `Main.registerAllExtensions`.

Up to you whether to put this in a subpackage or keep it here.

Reply:
Ok